routes/redis/GetState.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * Created by Kun on 6/28/2015.
+ */
+
+
+
+var express = require("express");
+var Q = require('q');
+var router = express.Router();
+var redis = require('redis');
+var config = require("konfig")();
+
+var port=config.app.redis.port;
+var host=config.app.redis.host;
+
+var client = redis.createClient(port,host);
+
+
+router.route('/:cookie').get(function(req, res) {
```

Comment:
We should be using cookie parser to get the cookie, not a route param, the frontend wont be parsing their cookie, it will just come in a header. Check out Elijah's branch for how to get the cookies.